assets/src/dashboard/style.css

```diff
@@ -38,6 +38,7 @@ body.web-story_page_stories-dashboard #wpbody-content {
 body.web-story_page_stories-dashboard #web-stories-dashboard {
   position: relative;
   overflow: hidden;
+  height: calc(100vh - 32px); /* ADMIN_TOOLBAR_HEIGHT = 32 */
```

Comment:
This mirrors how the editor currently calculates the total height.

Reply:
Should admin_toolbar_height be a constant? is this the only place we ever need that height?

Reply:
I could set this as a css variable but it will still be limited to these files. I think it would be great to find a way to determine the height of the Wordpress chrome.

Reply:
fair enough, this is fine!